### DISCUSSION TITLE
'source init_floorplan1.tcl'/ 'source init_floorplan2.tcl'   doesn't match the decumentation results .

### DISCUSSION THREAD

**Discussion by ReemAbuRajab7** [The-OpenROAD-Project/OpenROAD#6061] [Q&A] [answered]
![Screenshot from 2024-10-30 15-04-10](https://github.com/user-attachments/assets/1697f272-4335-4d07-871c-97e1ecdc0602)

**Comment by ReemAbuRajab7** [comment]
![111](https://github.com/user-attachments/assets/27185c26-b5b0-4339-b766-41c8b3e63fe2)

**Comment by ReemAbuRajab7** [comment]
![;;;](https://github.com/user-attachments/assets/e74f96f9-664f-4476-8daa-a7bf1252f15e)

**Comment by ReemAbuRajab7** [comment, marked as answer]
**solved** 
 source init_floorplan2.tcl --> turn on the rows 
source init_floorplan1.tcl --> turn on the heatmap

**Reply by shimoshida** [reply]
The floorplan2 was displayed successfully, but the green box was not displayed on the floorplan1. I have heatmap turned on...

**Comment by maliberty** [comment]
Is this for documenting your investigations or is there an unanswered question here?

**Reply by ReemAbuRajab7** [reply]
I was answering but solve it but didn't know how to take the discussion back .

**Reply by maliberty** [reply]
No problem maybe someone else will find it useful.
